reliability.md

```diff
@@ -1,33 +1,45 @@
 # Reliability
 
-Once your site is up, how do you keep it that way? And what can you do if it goes down? In most cases, the same best practices apply to WordPress as with other web applications, but some differences and recommendations are detailed here.
+When we already have a site in operation, the work of hosting does not end. The machines can and do fail, so we need to have methods to check that it is working as well as a backup system.
 
-## Backups
+## Backup
 
-A WordPress site is composed of three (3) main components: Code WordPress core, zero (0) or more plugins, and one (1) or more themes Assets Typically images, documents and other user-upload files. This can also contain plugin or theme cache and/or configuration files as well. Database A set of tables containing your posts, pages, comments, links, settings, and other data.
+WordPress does not have any automatic backup system by default, so you have to rely on a third-party tool to do so.
 
-> Your WordPress database contains user-generated content and configuration, including every post, every comment and every link you have on your website. If your database gets erased or corrupted, you stand to lose everything you have written. There are many reasons why this could happen and not all are things you can control. With a proper backup of your WordPress database and files, you can quickly restore things back to normal.
+When making a backup, you must bear in mind that WordPress is made up of two completely different elements: the files and the data. The file part is stored on your server, and can be accessed by FTP or any other method, and the data is stored in the database. This means that when you make a backup you have to do it from both parts, since one does not work without the other.
 
-It's recommended to keep and test regular backups of your WordPress sites using the system-level backup or snapshot infrastructure of your choice. One thing to be aware of is that the code, assets and database change in conjunction with WordPress but may be backed up separately. For this reason, it is a good idea to keep restore points that include backups of code, assets, and database taken at the same point in time. It’s recommended to create restore points before any critical action, e.g. WordPress core update.
+### Where to make copies
 
-## Monitoring
+There are many methods for making backups. The first is the hosting server itself. Usually the providers themselves offer the possibility of making a copy of all the contents hosted on the machine. Usually these copies are made externally on a specific backup server. In this case, you should know how it works and when it is done, as well as how to restore the copy, if it can be partial, total, and if you have access to the data in a simple way.
```

Comment:
"The providers" in this case are the readers of the post. Maybe phrase this in a way that is hosting-provider-centric?